watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskFragment.java

```diff
@@ -189,12 +199,18 @@ public void onResponse(ArrayList<Task> tasks) {
                         setAllTasks(tasks);
                         mAllTaskAdapter.notifyDataSetChanged();
                         for(int i=0; i < mAllTaskAdapter.getGroupCount(); i++) mListView.expandGroup(i);
+
                     } else {
                         hideList();
                     }
                 }
-
-                if (mSwipeLayout != null) mSwipeLayout.setRefreshing(false);
+                    new CountDownTimer(2000, 1000) {
+                        public void onTick(long millisUntilFinished) {}
+                        public void onFinish() {
+                            if (mSwipeLayout != null) mSwipeLayout.setRefreshing(false);
+                            if (mNoTasks != null) mNoTasks.setRefreshing(false);
+                        }
+                    }.start();
```

Comment:
to keep the layout from spazzing out if theres no data to be loaded....it like flashes everytime